x-pack/legacy/plugins/actions/server/builtin_action_types/server_log.ts

```diff
@@ -48,8 +52,9 @@ async function executor(
   const actionId = execOptions.actionId;
   const params = execOptions.params as ActionParamsType;
 
+  const sanitizedMessage = withoutControlCharacters(params.message);
   try {
-    logger[params.level](params.message);
+    logger[params.level](`server-log: ${sanitizedMessage}`);
```

Comment:
Maybe 'server-log' should be pulled out as a symbol, as it is also used in the action type name and id?

Reply:
makes sense